libsrc/sym1/bitio.s

```diff
@@ -0,0 +1,230 @@
+; ---------------------------------------------------------------------------
+; bitio.s
+;
+; for Sym-1
+;
+; Wayne Parham
+;
+; wayne@parhamdata.com
+; ---------------------------------------------------------------------------
+
+.include        "sym1.inc"
+
+.export         _set_DDR1A, _get_DDR1A, _set_IOR1A, _get_IOR1A
+.export         _set_DDR1B, _get_DDR1B, _set_IOR1B, _get_IOR1B
+.export         _set_DDR2A, _get_DDR2A, _set_IOR2A, _get_IOR2A
+.export         _set_DDR2B, _get_DDR2B, _set_IOR2B, _get_IOR2B
+.export         _set_DDR3A, _get_DDR3A, _set_IOR3A, _get_IOR3A
+.export         _set_DDR3B, _get_DDR3B, _set_IOR3B, _get_IOR3B
```

Comment:
I like using getters and setters.  I use PEEK and POKE in BASIC, but I prefer getters and setters in C.  More "C-Like."  :-)

Reply:
Rather "C++ overhead" IMHO.

Reply:
You got me wrong. I propose that you implement your getters and setters as macros using PEEK and POKE internally.

Reply:
Independently from the question how the getters and setters are implemented my first question is not answered yet: Is it really desirable to have many individual getters and setter? I imagine that it might very well be preferable to have just one getter and one setter with the register to get/set being a parameter. And to have a `#define` for every register name:

```
set(DDR1A, val);
val = get(DDR1A);
```

Reply:
Good suggestions.  I've modified sym1.h to use structs instead of getters and setters.